### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by nhennigan**
Update to list of Intel supported NICs and drivers. Linting on both files also.

### CODE REVIEW

**Review by Eoghan1232 — APPROVED** [review]

File: docs/vf-setup.md [hunk 1]
```diff
@@ -1,68 +1,83 @@
...
-	* Intel® Ethernet Controller X710
-* **Intel ixgbe Driver** 
-	* Intel® Ethernet Controller 82599
-	* Intel® Ethernet Controller X520
-	* Intel® Ethernet Controller X540
-	* Intel® Ethernet Controller x550
-	* Intel® Ethernet Controller X552
-	* Intel® Ethernet Controller X553
+  
+  * Intel® Ethernet Network Adapter X722
```
Comment (SchSeba) [inline docs/vf-setup.md:17]: can you please also add a list for the E810 supported nics?

**Review by SchSeba — COMMENTED** [review]
This is great thanks for the contribution!

btw k8s < 1.15 is not supported any more will it be possible to remove it from the docs?
I am fine if you want to do it in a separe PR

File: README.md [hunk 2]
```diff
@@ -58,21 +64,21 @@ To deploy workloads with SR-IOV VF or PCI PF, this plugin needs to work together
...
- - Intel® 82599ES
- - Mellanox ConnectX-4®
- - Mellanox Connectx-4® Lx EN Adapter
- - Mellanox ConnectX-5®
- - Mellanox ConnectX-5® Ex
- - Mellanox ConnectX-6®
- - Mellanox ConnectX-6® Dx
+
+- Intel® Ethernet 800 Series (E810)
+- Intel® Ethernet 700 Series (XL710, X710, XXV710, X72
```
Comment (zeeke) [inline README.md:74]: nit:

```suggestion
- Intel® Ethernet 700 Series (XL710, X710, XXV710)
```

X722 seems to to not be supported yet:
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator/templates/configmap.yaml#L6

**Comment by nhennigan** [comment]
> possible to remove it from the docs?
@SchSeba 
No problem, I can remove the line pointing to deployments/k8s-v1.10-v1.15/sriovdp-daemonset.yaml for 1.15 k8s support. Is there also a need then to remove this file in this PR?

**Comment by SchSeba** [comment]
Hi @nhennigan yes please remove the files if we don't point to them we don't need them :)

File: README.md (hunk 2 above)
Comment (nhennigan) [reply README.md]: I am enquiring about support for X722. I have removed it for now from the README but will create a new PR to add it if it is in fact supported.

File: docs/vf-setup.md (hunk 1 above)
Comment (nhennigan) [reply docs/vf-setup.md:31]: Added E810 NICs and also removed line about support for k8s <1.15

File: README.md (hunk 2 above)
Comment (zeeke) [reply README.md]: Thanks

**Pull request merged**